### ISSUE TITLE
Placeholder on Text items

### ISSUE THREAD

**Issue by genachka** [swisnl/jQuery-contextMenu#290]
Is it possible to add a placeholder text for items that are of type Text?

**Comment by bbrala** [comment]
No, currently you cannot define this when defining the items. You could use the show event (http://swisnl.github.io/jQuery-contextMenu/docs.html#events) or another callback, perhaps of the item itself, to change the dom to have a callback.

Maybe this could be an option in the items object. But for now, you need to use callbacks.

**Comment by genachka** [comment]
Can you please flag this as a feature request.
